src/main/scala/io/iohk/ethereum/ledger/Ledger.scala

```diff
@@ -57,8 +57,6 @@ class LedgerImpl(vm: VM, blockchainConfig: BlockchainConfig) extends Ledger with
     val blockchain = BlockchainImpl(storages)
 
     val blockExecResult = for {
-      //todo why do we require in executeBlockTransactions separate blockchain and blockchain storage
-      //todo blockchain storage is part of blockchain
```

Comment:
While looking at this I also realized that we're using `BlockchainStorages` and `BlockchainImpl` in several places of our code, maybe it should be better to rather work with `Blockchain`, as we are mixing the implementation and the abstraction of our `Blockchain` as in here? If so, we should create a task to look into this, and re-add this TODO.